homeassistant/components/vacuum/__init__.py

```diff
@@ -384,6 +391,18 @@ def turn_off(self, **kwargs):
         await self.hass.async_add_executor_job(
             partial(self.turn_off, **kwargs))
 
+    def start_pause(self, **kwargs):
```

Comment:
Are there any tests for this method and service now that we've moved it from base to legacy vaccum entity class.